app/routes.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+var	express = require('express');
+var	config = require('./controller/settings');
+
+var	router = express.Router();
+
+// Route to UPDATE the config
+router.post('/configs', function (req, res) {
+	var configData = config.update(req);
```

Comment:
for example reg.params.update = config.update(req). then you can just pass it along as part of the req object with res.send